.gitignore

```diff
@@ -127,3 +127,7 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+
+#kate
+*.kate-swp
```

Comment:
> 100644 → 100755

I don't think the file should have the execution bit.

Reply:
done